cli/list.go

```diff
@@ -28,7 +28,7 @@ func (l list) Run(flags GlobalFlags) error {
 	if err != nil {
 		return fmt.Errorf("failed to load triggers: %w", err)
 	}
-	log.Info("Available triggers:\n\n")
+	slog.Info("Available triggers:\n\n")
```

Comment:
Handling of newlines depends on the handler: [tint keeps them in the middle of the line](https://go.dev/play/p/_TQS-9kq5Fi), but the [text handler escapes them](https://go.dev/play/p/WyrTmJMvzKQ). Maybe we should just remove all trailing newlines?

```suggestion
	slog.Info("Available triggers:")
```